core/argument_parser.py

```diff
@@ -12,16 +12,32 @@
         'zura': 'Kunikida Hanamaru',
         'technology': 'Kunikida Hanamaru',
         'woah': 'Kunikida Hanamaru',
-        'mike': 'Kurosawa Ruby',  # Memes
+        'mike': 'Kurosawa Ruby',  # Memes     Who the fuck is Mike
         'nell': "Yazawa Nico",
-        'nya': 'Hoshizora Rin'
+        'nya': 'Hoshizora Rin',
+        'shiny': 'Ohara Mari',
+        'shiny~!': 'Ohara Mari',
+        'lili': 'Sakurachi Riko',
+        'its': 'Ohara Mari',
+        'joke': 'Ohara Mari',
+        'buu': 'Kurosawa Dia',
+        'kayo-chin': 'Koizumi Hanayo',
+        'yousoro': 'Watanabe You',
+        'rice': 'Koizumi Hanayo',
+        'mirai': 'Kunikida Hanamaru',
+        'future': 'Kunikida Hanamaru',
+        'mikan': 'Takami Chika',
+        'я': 'Ayase Eli',
+        'best': 'Nishikino Maki'
```

Comment:
People actually try using this as a filter so it is important that we remain impartial in the best girl debate. Delete this.